app/assets/stylesheets/hoge/fuga.css.scss

```diff
@@ -0,0 +1,40 @@
+.cat{}
+
+.dog {
+  color: #ff00ff;
+}
+
+p {
+  background: #ab;
+}
+
+
+span.hoge {
+        padding: 10px;
+}
+
+.piyo {
+  margin: 0.5em;
+}
+
+.piyo {
+  padding: 10px;
+}
+
+.one {
+  .two {
+    .three {
+      .four {
```

Comment:
__[scss_lint]__
* [SelectorDepth] Selector should have depth of applicability no greater than 3, but was 4
* [NestingDepth] Nesting should be no greater than 3, but was 4

<!-- Comment by [SideCI](https://sideci.com) -->